packages/fiori-elements-writer/test/__snapshots__/lrop.test.ts.snap

```diff
@@ -21954,25 +21954,29 @@ sap.registerComponentDependencyPaths(manifestUri)
     "contents": "<!DOCTYPE html>
 <html>
   <head>
-    <title>QUnit test suite</title>
+    <title>
```

Comment:
Why are there changes in FE templates when the new feature is to support FF tests?

Reply:
Thx